sdk/include/reactos/undocuser.h

```diff
@@ -296,6 +296,12 @@ MessageBoxTimeoutW(
 
 LPCWSTR WINAPI MB_GetString(IN UINT wBtn);
 
+/* dwType for NtUserUpdateInputContext */
+enum UPDATE_INPUT_CONTEXT
```

Comment:
Not `typedef enum _UPDATE_INPUT_CONTEXT { ... } UPDATE_INPUT_CONTEXT;` ?

Reply:
Fixed.